win32ss/gdi/dib/dib16bpp.c

```diff
@@ -353,42 +669,84 @@ DIB_16BPP_BitBltSrcCopy(PBLTINFO BltInfo)
         *((WORD *)DestBits) = (WORD)XLATEOBJ_iXlate(
           BltInfo->XlateSourceToDest, xColor);
 
-        SourceBits += 3;
+        if (bLeftToRight)
+        {
+          SourceBits -= 3;
+        }
+        else
+        {
+          SourceBits += 3;
+        }
```

Comment:
For example with the suggested increment macro from above, this could hold in one single line:
```c
    INCREMENT(SourceBits, bLeftToRight, 3);
```

Reply:
Fixed by changing this code the new DEC_OR_INC macro.